### PR TITLE
[WIP]Ports the TG achievement framework. Ports some achievements.

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Title.

## Why It's Good For The Game

Achievements are fun to unlock and to show off your hypothetical, virtual cock. This provides a framework to add achievements and adds some silly ones, including megafauna achievements

## Changelog
:cl:
add: Ported /tg/ achievement system.
add: Added some, but not all, of tg's achievements.
code: Changed some code for some achievements to be possible.
/:cl:

### CODE REVIEW

**Comment by Ghommie** [comment]
Based, and I was thinking of porting it sooner or later (and perhaps the SDQL migration too).

**Comment by BobJoga** [comment]
Lemme add the WIP tag since this isn't actually quite "working" yet.

**Comment by BobJoga** [comment]
@Ghommie Alright so, everything seems to be working... except that the check achievement OOC command just shows a blank page, and i have NO IDEA how to fix it.

File: tools/HubMigrator/HubMigrator.dm [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+						MINER_SCORE,
+						BUBBLEGUM_SCORE,
+						COLOSSUS_SCORE,
+						DRAKE_SCORE,
+						HIEROPHANT_SCORE,
+						LEGION_SCORE,
+						SWARMER_BEACON_SCORE,
+						TENDRIL_CLEAR_SCORE)
+
+	var/ach = "achievements" //IMPORTANT : ADD PREFIX HERE IF YOU'RE USING PREFIXED SCHEMA
```
Comment (silicons) [inline tools/HubMigrator/HubMigrator.dm:91]: @LetterJay

**Comment by BobJoga** [comment]
might throw in the towel tbh this shit ain't working

**Pull request closed without merging**